swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -29,16 +32,28 @@ enum LogLevel {
     case assert, error, warning, info, debug
 }
 
+/// For the hardcoded values that represent the privacy settings we have an example of more enum utilization here.
 enum PrivacySetting: String {
     case exclude = "fs-exclude"
     case mask = "fs-mask"
     case unmask = "fs-unmask"
 }
 
+/**
+ This is a convenience method that takes a case and a dictionary to title the event and provide properties.
```

Comment:
Up to you on how to approach, but generally I try keep comments as straightforward as possible (fewest amount of words to convey the required info).  Reading the current description, it's not clear that this eventually calls `FS.event`.